src/vmm/src/arch/aarch64/registers.rs

```diff
@@ -0,0 +1,662 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Nit: It looks like we name similar files `regs.rs`, not `registers.rs`.

Reply:
renamed it back to `regs.rs`